app/views/layouts/application.html.erb

```diff
@@ -17,9 +17,10 @@
     <%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %>
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
+    
     <%= hotwire_livereload_tags if Rails.env.development? %>
 
-    <%= turbo_refreshes_with method: :morph, scroll: :preserve %>
```

Comment:
Hey @sarvaiyanidhi, we definitely want to keep `turbo_refreshes_with method: :morph, scroll: :preserve`. Morphing is a really cool feature of Turbo 8 that's we're going to want to utilize.

Is it possible to fix this issue with just the `config.action_view.preload_links_header` change?

Reply:
Thanks @tjbarber for feedback. 

By just adding `config.action_view.preload_links_header` fixes Tailwind changes not reflecting every time on file change but theme issue still persist.

I believe theme old value is not getting retained due to morph code. I understand that we need this feature for the application but on checking further it seems like it is already identified issue. Came across related issues on the same https://github.com/hotwired/turbo/issues/1210

Reply:
Hey @sarvaiyanidhi ! We talked this one over and we think it's best to actually pursue an entirely different solution for theme switching altogether. We're considering options like persisting the state on some sort of `user_settings` table that we still have to map out. Thanks for putting time into this! Would you close this PR? I will close the issue as obsolete for now.

Reply:
Noted. Will close this PR.